ui/app/serializers/identity/entity-alias.js

```diff
@@ -4,4 +4,27 @@
  */
 
 import IdentitySerializer from './_base';
-export default IdentitySerializer.extend();
+export default IdentitySerializer.extend({
+  extractLazyPaginatedData(payload) {
+    return payload.data.keys.map((key) => {
+      const model = payload.data.key_info[key];
```

Comment:
Used the same logic that's in the IdentitySerializer's normalizeItems method so we still have metadata on the list